File: Baekjoon/Greedy/2217.py
# https://www.acmicpc.net/problem/2217

n = int(input())
k = []
answers = []

for _ in range(n):
    k.append(int(input()))
k.sort()

for x in k:
    answers.append(x * n)
    n -= 1

print(max(answers))

File: Baekjoon/Greedy/2217.cpp
// https://www.acmicpc.net/problem/2217

#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;

int main() {
    int N;
    cin >> N;
    vector<int> A(N);
    for (int i = 0; i < N; i++) {
        cin >> A[i];
    }
    sort(A.begin(), A.end());
    int ans = 0;
    for (int i = 0; i < N; i++) {
        ans = max(ans, A[i] * (N - i));
    }
    cout << ans << '\n';

    return 0;
}